packages/react-native/src/primitives/Checkbox/types.ts

```diff
@@ -1,13 +1,11 @@
 import { StyleProp, ViewStyle } from 'react-native';
-
-import { IconButtonProps } from '../IconButton';
-
+import { IconProps } from '../Icon/types';
 import { LabelProps } from '../Label';
 import { LabelPosition } from '../Label/types';
 
-export interface CheckboxProps<T>
-  extends Omit<IconButtonProps, 'source' | 'style'> {
-  buttonStyle?: IconButtonProps['style'];
+export interface CheckboxProps<T> extends Omit<IconProps, 'source' | 'style'> {
```

Comment:
This is might be out of scope for this PR but maybe we should readdress the `CheckboxProps`. Thinking that they should extend from `PressableProps`, what do you think?

Reply:
Thanks for bringing this up, I thought it looked weird. Addressing now.